pubsub/gossipsub/README.md

```diff
@@ -298,35 +305,29 @@ for each topic in mesh:
      emit PRUNE(topic) control message to peer
 ```
 
-The `fanout` map is maintained by keeping track of last published time
-for each topic:
+The `fanout` map is maintained by keeping track of the length of time since a 
+topic was last published:
 ```
 for each topic in fanout:
- if last published time > TTL
-   remove topic from fanout
- else if |fanout[topic]| < D
-   select D - |fanout[topic]| peers from peers.gossipsub[topic] - fanout[topic]
-   add the peers to fanout[topic]
+  if time_since_last_published > ttl
+    remove topic from fanout
+  else if |fanout[topic]| < D
+    select D - |fanout[topic]| peers from peers.gossipsub[topic] - fanout[topic]
+    add the peers to fanout[topic]
 ```
 
 Gossip is emitted by selecting peers for each topic that are not already part
 of the mesh:
 ```
 for each topic in mesh+fanout:
-  let mids be mcache.window[topic]
-  if mids is not empty:
+  let m_ids be mcache.window[topic]
```

Comment:
ugh, please don't. That's inconsistent naming.

Reply:
here, please no underscores! 
They are such an artifact of common programming language limitations and lack of symbolic freedom.